ChocolateyPackage/textanalysistoolnet.portable.nuspec

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>textanalysistoolnet.portable</id>
+    <version>2018.11.20</version>
+    <owners>bcurran3</owners>
+    <packageSourceUrl>https://github.com/TextAnalysisTool/Releases/tree/master/ChocolateyPackage</packageSourceUrl>
+    <title>TextAnalysisTool.NET (Portable)</title>
+    <authors>David Anson</authors>
+    <projectUrl>https://github.com/TextAnalysisTool/Releases</projectUrl>
+    <iconUrl>https://github.com/TextAnalysisTool/TextAnalysisTool.github.io/blob/master/TextAnalysisTool.png</iconUrl>
+    <copyright>© David Anson</copyright>
+    <licenseUrl>https://opensource.org/licenses/MIT</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <docsUrl>https://textanalysistool.github.io/TextAnalysisTool.NET.txt</docsUrl>
+	<bugTrackerUrl>https://github.com/TextAnalysisTool/Releases/issues</bugTrackerUrl>
+    <tags>text analysis tool .net</tags>
+    <summary>TextAnalysisTool.NET is a free program designed to excel at viewing, searching, and navigating large files quickly and efficiently. TextAnalysisTool.NET provides a view of the data that you can easily manipulate (through the use of various filters) to display exactly the information you need - as you need it!</summary>
+    <description>
```

Comment:
Is there a way to link to this instead of embedding the document?

Reply:
I can add a link to this page as well, but this serves as a base to what gets shown on Chocolatey's page (see https://chocolatey.org/packages/notepadplusplus for example)